Taskfile.yml

```
@@ -0,0 +1,163 @@
+tasks:
+
+  install-changie:
+    desc: Install changie
+    aliases: [get-changie, get-changie]
+    cmds:
+      - echo "Installing changie..."
+      - task: install-changie-mac-os
+      - echo -e "\nSuccess! 'terraform' installed."
+
+  install-changie-mac-os:
+    internal: true
+    platforms: [darwin]
+    cmds:
+      - brew tap miniscruff/changie https://github.com/miniscruff/changie
+      - brew install changie
+    preconditions:
+      - sh: 'which brew'
+        msg: '"brew" needed to install changie - see https://brew.sh'
+
+  install-terraform:
+    desc: Install terraform
+    aliases: [get-tf, get-terraform]
+    cmds:
+      - echo "Installing terraform..."
+      - task: install-terraform-mac-os
+      - echo -e "\nSuccess! 'terraform' installed."
+    silent: true
+
+  install-terraform-mac-os:
+    internal: true
+    platforms: [darwin]
+    cmds:
+      - brew tap hashicorp/tap
+      - brew install hashicorp/tap/terraform
+    preconditions:
+      - sh: 'which brew'
+        msg: '"brew" needed to install terraform - see https://brew.sh'
+
+  lint:
+    desc: Formatting and linting go code
+    dir: "{{.OPSLEVEL_DIR}}"
+    cmds:
+      - gofmt -d .
+      - go vet ./...
+      - golangci-lint run
+
+  lintfix:
+    desc: Fix formatting and linting
+    dir: "{{.OPSLEVEL_DIR}}"
+    env:
+      TF_ACC: true
+    cmds:
+      - gofmt -w .
+      - go mod tidy
+      - golangci-lint run --fix
+
+  terraform-apply:
+    desc: Run terraform apply
+    aliases: [tf-apply]
+    dir: "{{.WORKSPACE_DIR}}"
+    cmds: [terraform apply]
+    preconditions:
+      - sh: 'which terraform'
+        msg: '"terraform" needed - run "brew install terraform"'
+
+  terraform-build:
+    desc: Build terraform
+    aliases: [tf-build]
+    platforms: [darwin]
+    vars:
+      BINARY: terraform-provider-opslevel_{{.VERSION_DYNAMIC}}
+      VERSION:
+        sh: echo "{{.BINARY}} | cut -d '_' -f 2"
+    cmds:
+      - go build -o {{.BINARY}} || exit 1
+      - chmod +x {{.BINARY}}
+      - mkdir -p {{.BINARY_PATH}}
+      - mv {{.BINARY}} {{.BINARY_PATH}}
+      - cmd: echo "Built terraform provider at:"
+        silent: true
+      - cmd: echo -e "  {{.BINARY_PATH}}/{{.BINARY}}"
+        silent: true
+
+  terraform-clean:
+    desc: Clean up workspace
+    aliases: [tf-clean]
+    cmds:
+      - rm -rf {{.BINARY-PATH}}
+
+  terraform-init:
+    desc: Run terraform init
+    aliases: [tf-init]
+    dir: "{{.WORKSPACE_DIR}}"
+    cmds:
+      - |
+          cat << EOF > backend.tf
+          terraform {
+            required_providers {
+              opslevel = {
+                source  = "OpsLevel/opslevel"
+                version = "> 0.0.1"
+              }
+            }
+          }
+
+          provider "opslevel" {}
+          EOF
+      - touch main.tf
+      - terraform init --upgrade
+    preconditions:
+      - sh: 'which terraform'
+        msg: '"terraform" needed - run "brew install terraform"'
+
+  terraform-workspace-init:
```

how about 
```suggestion
  terraform-build-init:
```

with alias `tf-binit` ?